Makefile

```diff
@@ -40,7 +40,8 @@ web-resources:
 	cp shared/data/web_accessible_resources/* build/$(browser)/$(type)/web_accessible_resources/
 	cp shared/tracker-surrogates/surrogates/*.js build/$(browser)/$(type)/web_accessible_resources/
 	mkdir -p build/$(browser)/$(type)/data
-	node scripts/generateListOfSurrogates.js -i build/$(browser)/$(type)/web_accessible_resources/ >> build/$(browser)/$(type)/data/surrogates.txt
```

Comment:
I don't think we use this .txt file at all (perhaps in tests).

Does https://github.com/duckduckgo/tracker-surrogates/blob/main/mapping.json serve the same purpose? I think we could import that as other clients do also.

Reply:
Ignore that we're not using it processSurrogateList is in privacy-grade.

However I wonder if consuming the mapping.json would be cleaner?